packages/react-core/src/components/Button/examples/Button.md

```diff
@@ -52,18 +52,46 @@ import { TimesIcon, PlusCircleIcon } from '@patternfly/react-icons';
 
 LinkButton = () => (
   <React.Fragment>
-    <Button component="a" href="https://pf-next.com/" target="_blank" variant="primary">
-      Link to Core Docs
+    <Button component="a" href="https://pf4.patternfly.org/" target="_blank" variant="primary">
+      Link to core docs
     </Button>{' '}
-    <Button component="a" href="https://pf-next.com/" target="_blank" variant="secondary">
-      Secondary Link to Core Docs
+    <Button component="a" href="https://pf4.patternfly.org/" target="_blank" variant="secondary">
+      Secondary link to core docs
     </Button>{' '}
-    <Button component="a" isDisabled href="https://pf-next.com/" target="_blank" variant="tertiary">
-      Tertiary Link to Core Docs
+    <Button component="a" isDisabled href="https://pf4.patternfly.org/" target="_blank" variant="tertiary">
+      Tertiary link to core docs
     </Button>
     <Button component="a" href="https://pf4.patternfly.org/contribution/#modifiers" variant="link">
       Jump to modifiers in contribution guidelines
     </Button>
   </React.Fragment>
 );
 ```
+
+```js title=Aria-disabled-button-with-tooltip
+import React from 'react';
+import { Button } from '@patternfly/react-core';
+import { TimesIcon, PlusCircleIcon, Tooltip } from '@patternfly/react-icons';
+
+AriaDisabledBtnWithTooltip = () => (
+  <Tooltip content="Aria-disabled buttons are like disabled buttons, but focusable. Allows for tooltip support.">
+    <Button isAriaDisabled variant="secondary" onKeyPress={() => {alert('KeyPress')}} onClick={() => {alert('Click')}}>
```

Comment:
Why have `onKeypress` and `onClick` here if the button `isAriaDisabled`, is this leftover for testing? These events should not function, correct?

Reply:
Yes exactly, it was there to show the events are correctly being suppressed. I can move it over into react-integration and just make sure it's covered from a test perspective. Good catch!